src/vt/context/runnable_context/papi_data.h

```diff
@@ -0,0 +1,154 @@
+//@HEADER
+*/
+
+#if !defined INCLUDED_VT_CONTEXT_RUNNABLE_CONTEXT_LB_DATA_PAPI_DATA_H
+#define INCLUDED_VT_CONTEXT_RUNNABLE_CONTEXT_LB_DATA_PAPI_DATA_H
+
+#include <papi.h>
+#include INCLUDE_FMT_CORE
+
+namespace vt { namespace ctx {
+
+/**
+ * \struct PAPIData
+ *
+ * \brief Structure for storing Performance API (PAPI) related data structures
+ */
+struct PAPIData {
+  int EventSet = PAPI_NULL;
+  int retval = 0;
+  uint64_t start_real_cycles = 0, end_real_cycles = 0, start_real_usec = 0, end_real_usec = 0;
+  uint64_t start_virt_cycles = 0, end_virt_cycles = 0, start_virt_usec = 0, end_virt_usec = 0;
+  std::vector<std::string> native_events = {};
+  std::vector<uint64_t> values = {};
+
+  PAPIData()
+  {
+    const char* env_p =  getenv("VT_EVENTS");
+
+    // check if the environment variable is set
+    if (env_p == nullptr) {
+      fmt::print("Warning: Environment variabale VT_EVENTS not set, defaulting to instructions for the PAPI event set.\n");
+      native_events.push_back("PAPI_TOT_INS");
+    }
+    else {
+      std::string env_str(env_p);
+
+      std::stringstream ss(env_str);
+      std::string item;
+
+      while (std::getline(ss, item, ','))
+      {
+        native_events.push_back(item);
+      }
+    }
+    values.resize(native_events.size());
+    std::fill(values.begin(), values.end(), 0);
+
+    /* Create an EventSet */
+    retval = PAPI_create_eventset(&EventSet);
+    if (retval != PAPI_OK)
+      handle_error("PAPIData Constructor: Couldn't create an event set: ");
+
+    /* Add one event to our EventSet,
+    must be done such that we can call PAPI_set_multiplex */
+    int native = 0x0;
+    retval = PAPI_event_name_to_code(native_events[0].c_str(), &native);
+    retval = PAPI_add_event(EventSet, native);
+    if (retval != PAPI_OK)
+      handle_error("PAPIData Constructor: Couldn't add instructions to event set: ");
+
+    /* Convert the EventSet to a multiplexed EventSet */
+    retval = PAPI_set_multiplex(EventSet);
+    if (retval != PAPI_OK)
+      handle_error("PAPIData Constructor: Couldn't convert event set to multiplexed: ");
+
+    // Starting at i=1 because we've already added the first event
+    for (size_t i = 1; i < native_events.size(); i++) {
+      native = 0x0;
+      retval = PAPI_event_name_to_code(native_events[i].c_str(), &native);
+      if (retval != PAPI_OK) {
+        vtAbort(fmt::format("Couldn't event_name_to_code for {}: PAPI error {}: {}\n", native_events[i].c_str(), retval, PAPI_strerror(retval)));
+      }
+      retval = PAPI_add_event(EventSet, native);
+      if (retval != PAPI_OK) {
+        vtAbort(fmt::format("Couldn't add {} to the PAPI Event Set: PAPI error {}: {}\n", native_events[i].c_str(), retval, PAPI_strerror(retval)));
+      }
+    }
+  }
+
+  void handle_error (std::string info) const
+  {
+    vtAbort(fmt::format("{}: PAPI error {}: {}\n", info.c_str(), retval, PAPI_strerror(retval)));
+  }
+
+  void start()
+  {
```

Comment:
```suggestion
  void start() {
```
Function's opening bracket [should be placed in the same line](https://github.com/DARMA-tasking/vt/wiki/Code-Style-Guidelines). Update other functions as well.

Reply:
BTW there's a `.clang-format` file available in the repository. It doesn't reflect vt's preferred style ideally, but for most cases it should be close enough.